go.mod

```diff
@@ -66,12 +66,12 @@ require (
 	github.com/hashicorp/go-secure-stdlib/gatedwriter v0.1.1
 	github.com/hashicorp/go-secure-stdlib/kv-builder v0.1.2
 	github.com/hashicorp/go-secure-stdlib/nonceutil v0.1.0
-	github.com/hashicorp/go-secure-stdlib/parseutil v0.1.9
+	github.com/hashicorp/go-secure-stdlib/parseutil v0.2.0
 	github.com/hashicorp/go-secure-stdlib/password v0.1.1
 	github.com/hashicorp/go-secure-stdlib/reloadutil v0.1.1
 	github.com/hashicorp/go-secure-stdlib/strutil v0.1.2
 	github.com/hashicorp/go-secure-stdlib/tlsutil v0.1.2
-	github.com/hashicorp/go-sockaddr v1.0.6
+	github.com/hashicorp/go-sockaddr v1.0.7
```

Comment:
This change at least makes sense ,though I'm slightly confused where the `parseutil` change comes from. 

You might need to run `make sync-deps` so `api/` and `sdk/` are updated.

Reply:
I don't know either for the parseutil change. I tried to rollback to v0.1.9 but go mod tidy keeps upgrading them.

`go mod why` does not give more insight